README.md

```diff
@@ -1,2 +1,66 @@
 # Paseo Github repository
+Welcome to the Paseo testnet repo!
```

Comment:
```suggestion
Welcome to the Paseo testnet!
```

Reply:
we might say as well "Welcome to the Paseo Tesnet Organization". This is the readme for the whole organization and not the repo.